src/style.css

```diff
@@ -1,3 +1,93 @@
+/* stylelint-disable no-duplicate-selectors */
+html {
+  background-color: #e5e5e5;
+
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+
+}
+
+.header {
+  height: 60px;
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #ffff;
+}
+
+.logo {
+  display: block;
+  margin: 0 px;
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  height: inherit;
+  margin: 0;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  height: inherit;
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+}
+
+.nav__item {
+  display: flex;
+  align-items: center;
+  margin: 0 20px 0 0;
+  height: 100%;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  display: flex;
+  line-height: 60px;
+  font-family: Roboto, sans-serif;
```

Comment:
Consider using the common font for the entire body tag to get your elements inherited from it